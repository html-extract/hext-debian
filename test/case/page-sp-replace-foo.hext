<p @text:replace(/foo/, "bar"):bar_text />
