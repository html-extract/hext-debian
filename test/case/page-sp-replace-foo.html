<p>This is a foo containing foos</p>
