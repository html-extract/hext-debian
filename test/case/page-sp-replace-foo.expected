{"bar_text":"This is a bar containing bars"}
